cmd/incusd/api_project.go

```diff
@@ -1541,3 +1566,93 @@ func projectValidateRestrictedSubnets(s *state.State, value string) error {
 
 	return nil
 }
+
+// Do I need to check the StatusCode and/or ExpandedConfig similar to "cmd/incus/delete.go"
+// Do I need to check if this a snapshot or of type container similar to "cmd/incusd/instance_delete.go"
```

Comment:
GetInstances() should only return you instances so that should be fine, their snapshots will go away with them.